### PR TITLE
sachin first test commit

### PR DESCRIPTION
**Pull request by smpawar**
Just a small correction as first  test commit.
No major change

### CODE REVIEW

**Review by smpawar — COMMENTED** [review]
review ready

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ opdb__dbservicesinfo__122_0.1.1_oracle12c.ORCL.orcl.080421224807.log
 *  In this example of a filename `opdb__usedspacedetails__121_0.1.0_mydbhost.mycompany.com.ORCLDB.orcl1.071621111714.log` the file identification is `071621111714`.
 *  `-fileslocation`: The location in which the opdb*log were saved.
 *  `-projectname`: The GCP project in which the data will be loaded.
-*  `-deletedataset`: This an optinal. In case you want to delete the whole existing dataset before importing the data. 
+*  `-deletedataset`: This an optional. In case you want to delete the whole existing dataset before importing the data.
```
Comment (ashokhollav) [inline README.md:119]: should be "This is optional"

Comment (smpawar) [reply README.md]: done

**Comment by ashokhollav** [comment]
approved changes to readme.md

**Pull request closed without merging**